### PR TITLE
Add blog post for October 2015 Extravaganza.

### PR DESCRIPTION
**Pull request by Osmose**
@bobsilverberg @mathjazz @jotes @peterbe @erikrose @jgmize @fwenzel r?

I'll be posting this on Monday morning, so please leave any feedback or corrects by then. Thanks!

### CODE REVIEW

**Comment by mathjazz** [comment]
r+

File: blog/extravaganza/2015-10-October.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+[leaderboard]: https://pontoon.mozilla.org/contributors/
+
+### Air Mozilla iTunes Video Podcast
+[Peterbe][] stopped by to share the news that [Air Mozilla][] now has an
+iTunes-compatible [podcast feed][] for all of its videos. The feed has already
+been approved by Apple and is available on the iTunes Store.
+
+[Peterbe]: https://mozillians.org/en-US/u/peterbe/
+[Air Mozilla]: https://air.mozilla.org/
+[podcast feed]: https://air.mozilla.org/feed/itunes
```
Comment (peterbe) [inline blog/extravaganza/2015-10-October.md]: Add a slash so it's `https://air.mozilla.org/feed/itunes/` please

**Comment by peterbe** [comment]
r+ with nit

File: blog/extravaganza/2015-10-October.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+[localForage]: https://github.com/mozilla/localForage
+[localStorage]: https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage
+[blog post]: http://www.peterbe.com/plog/localstorage-is-fast
+
+### Mozilla.org Hosted on Deis
+Next was [jgmize][] sharing the news that [mozilla.org][] is now hosted on a
+[Deis][] cluster. Currently a small amount of production data is being hosted by
+the cluster, but a larger rollout is planned for the near future. [Giorgos][]
+and [pmac][] were also involved, and helped with a
+[Jenkins deployment pipeline display][], moving Apache redirects to Python, and
```
Comment (jgmize) [inline blog/extravaganza/2015-10-October.md]: @glogiotatidis deserves most of the credit for our entire Jenkins setup; the display is just the most visible part of what he worked on. He also wrote the [ansible playbook](https://github.com/mozilla/ee-infra-jenkins) used to configure the Jenkins server and the [scripts](https://github.com/mozilla/bedrock/tree/master/docker/jenkins) that execute in the deployment pipeline. 

@pmclanahan deserves the credit for slogging through thousands of lines of apache config and porting the redirects to python, and writing a comprehensive set of [tests](https://github.com/mozilla/bedrock/tree/master/test_redirects) for everything that was ported over.

**Comment by bobsilverberg** [comment]
r+

**Comment by jotes** [comment]
r+

On Fri, Oct 9, 2015 at 5:24 AM, Bob Silverberg <notifications@github.com>
wrote:

> r+
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/webdev/pull/27#issuecomment-146745645>.
>



-- 
jarek@reijutsu:~$ fortune
You have Egyptian flu: you're going to be a mummy.
jarek@reijutsu:~$ fortune
You now have Asian Flu.

**Pull request closed without merging**